src/types/transactions/index.ts

```diff
@@ -16,14 +16,13 @@ import {
   ApplicationClearStateTransaction as AppClearStateTxn,
   ApplicationNoOpTransaction as AppNoOpTxn,
 } from './application';
-
```

Comment:
One minor thing: could you add back this removed line? I can merge once that's done.

Reply:
done